src/components/PlayField/PlayField.tsx

```diff
@@ -0,0 +1,92 @@
+import styles from "./PlayField.module.css";
+import Card from "../Card/Card";
+import React from "react";
+import { initialImages } from "../../constants/initialImages";
+import { IOpenCardsItem, IPlayingFieldItem } from "../../types/types";
+import { shuffleArray } from "../../utils/util";
+
+const images = shuffleArray([...initialImages, ...initialImages]);
+const initialField: IPlayingFieldItem[] = images.map((url) => {
+  return {
+    url,
+    flip: false,
+    off: false,
+  };
+});
+
+function PlayField() {
+  const [openCards, setOpenCards] = React.useState<IOpenCardsItem[]>([]);
+  const [field, setField] = React.useState<IPlayingFieldItem[]>(initialField);
```

Comment:
What if you name this `[cards, setCards]`? Then `cards.map((card) => { ... }` would make more sense when you read it

Reply:
and `initialField` > `initialCards` then?